src/main/java/org/liquidengine/legui/binding/converter/AbstractClassConverter.java

```diff
@@ -0,0 +1,30 @@
+package org.liquidengine.legui.binding.converter;
+
+/**
+ * Class converter. Should be used to convert class to string representation and vise versa.
+ *
+ * @param <T> type of class converter.
+ * <p>
+ * Created by ShchAlexander on 02.12.2017.
+ */
+public abstract class AbstractClassConverter<T> {
```

Comment:
Okay, I'll replace it with interface :)

But what do you mean when you saying serializer and deserializer? Should it serialize to String or to some another type?

Reply:
I mean, the purpose of this class is to convert object to string, and string to object, so that can be broken in two interfaces, when I say ObjectSerializer I mean serialize object to string, and the ObjectDeserializer is the same but string to object, the inverse operation

Reply:
But why do you want to divide them?

Reply:
https://en.wikipedia.org/wiki/Interface_segregation_principle smaller interfaces, that do only one thing are usually better, the implementation can just add the 2 interfaces, in this case is not a big deal so it's fine using only 1 interface if you want.

Reply:
Discssed. Changed to interface.